ml_synth_multi_saw_example.ino

```diff
@@ -329,6 +335,9 @@ void loop_1Hz()
 #ifdef BLINK_LED_PIN
     Blink_Process();
 #endif
+#ifdef BOARD_TTGO_T9_RGB_LED_WM8978
```

Comment:
What do you think about to combine all in one Blink_Process function with a switch to select between LED via GPIO or WS2812b or similar?

Reply:
That was exactly what I was thinking. You have that YouTube clip where you are using a led strip. Well let's integrate WS2812b that as a feature. 

There must be a define for what LED on the string that should blink.
But with ledstrips it does not matter in cpu cycles if you blink or if you do something else with all leds. all leds are addressed even if one is updated.